app/views/upscan_csv_file_upload.scala.html

```diff
@@ -31,7 +31,7 @@
 @layout(
     applicationConfig = ac,
     title =  title,
-    backLink = Some(BackLinkParams("#"))
+    backLink = Some(BackLinkParams(href = "javascript:history.back()"))
```

Comment:
this was already being handled as a JS link here https://github.com/hmrc/gmp-frontend/blob/8df34d27333f0433df858e943315b31f0ba9e0df/app/views/Layout.scala.html#L63 because it was a hash link with "#" as the href.

Reply:
resolved in [09a0398](https://github.com/hmrc/gmp-frontend/pull/444/commits/09a03984924c929faa709339b444d60abbde44fd)